src/v/cluster/sloth_mail/README.md

```diff
@@ -0,0 +1,9 @@
+SlothMail is an RPC wrapper that batches non-urgent messages to reduce traffic.
+
+Each message comprises destination node id, mail kind, key, value and shipping deadline. Key and value types are specific to the mail kind. If a message with the same destination, mail kind and key is dispatched before the original is shipped they are merged with the kind-specific merge function. The earliest deadline is used.
```

Comment:
Let's stick to RPC request instead of a new name `mail`

Reply:
Functionally it's not an RPC request either, as no reply is returned to the sender.